charts/spire/profiles/production-values.yaml

```diff
@@ -1,3 +1,6 @@
+# NOTE: Do not edit the contents of this file. We will update it as we find
```

Comment:
This is an odd request to make.  Can you share the motivation behind the request.  With that, I can better review it.